api/elections.go

```diff
@@ -92,6 +92,14 @@ func (a *API) enableElectionHandlers() error {
 	); err != nil {
 		return err
 	}
+	if err := a.endpoint.RegisterMethod(
```

Comment:
If `chain/info` already returns this info, we don't need this new endpoint. I think we should try to avoid duplicate info. IMO, we can remove this new endpoint.

Reply:
but chain/info returns the number of transactions not the number of voting processes